scripts/update-index-alias.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+A script to update the search index an alias is assoiciated with.
+
+To delete the old index (the index losing the '<alias>-old' alias), set --delete-old-index=yes
+
+Usage:
+scripts/update-index-alias.py <alias> <target> <stage> <search-api-endpoint> [options]
+
+Options:
+    --delete-old-index=<delete-old-index>  [default: no]
+"""
+import os
+import sys
+import subprocess
+import yaml
+import json
+from distutils.util import strtobool
+import requests
+from requests.exceptions import HTTPError
+from docopt import docopt
+
+
+def update_index_alias(alias, target, stage, endpoint, delete_old_index):
+    auth_token = _get_auth_token(stage)
+    headers = {
+        'Authorization': "Bearer {}".format(auth_token),
+        'Content-type': 'application/json'
+    }
+
+    if delete_old_index:
+        old_index_to_delete = _get_old_index_to_delete(alias, endpoint)
+
+    _rename_current_alias_to_old(alias, endpoint, headers)
```

Comment:
The name is a bit misleading, as far as I can tell it's not renaming the current alias, it's *moving* the old one to match it.